src/main/scala/org/allenai/pdffigures2/FindGraphicsRaster.scala

```diff
@@ -57,7 +59,8 @@ object FindGraphicsRaster {
     pixThreshold: Int,
     rescale: Int
   ): List[Box] = {
-    val pixelsToExplore = scala.collection.mutable.Stack[Int]()
+    val pixelsToExplore = mutable.Set[Int]()
```

Comment:
That's a good question. But it ran out of memory before, and now it finishes in almost imperceptible time.